docs/tutorials/working_with_the_ensemble.ipynb

```diff
@@ -19,7 +19,7 @@
   },
```

Comment:
<p>grammar nit (and probably my fault): "if an TAPE..."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/tape/pull/290/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Done.</p>